net/tailscale/Makefile

```diff
@@ -28,7 +28,7 @@ GO_PKG:=\
 	tailscale.com/cmd/tailscaled
 GO_PKG_LDFLAGS:=-X 'tailscale.com/version.longStamp=$(PKG_VERSION)-$(PKG_RELEASE) (OpenWrt)'
 GO_PKG_LDFLAGS_X:=tailscale.com/version.shortStamp=$(PKG_VERSION)
-GO_PKG_TAGS:=ts_include_cli
+GO_PKG_TAGS:=ts_include_cli,ts_omit_aws,ts_omit_bird,ts_omit_tap,ts_omit_kube,ts_omit_completion
```

Comment:
However, you added commit description exactly the same as it is in the pull request, but still you have decided to build it with less features, you pointed us to https://github.com/tailscale/tailscale/blob/main/build_dist.sh#L40 and still I am not sure what was the reason to have fewer features. 

While looking at https://openwrt.org/submitting-patches, I see:
> It should explain to a competent reader why you made this commit.